files/anbox-app-manager.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Anbox Session Manager
+Requires=anbox-session-manager.service
+After=anbox-session-manager.service
+
+[Service]
+ExecStart=/snap/bin/anbox.appmgr
```

Comment:
The Snap for this is not actually installed. Due to the large installation size, I'd put the install of it in the first-time setup script under the productivity tools.

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/winesapos-setup.sh#L457